src/app/review-item/components/review-success-page/review-success-page.component.html

```diff
@@ -46,7 +48,8 @@ <h4>Erfahrungspunkte erhalten</h4>
                 </div>
                 <div class="xp__score">
                   <i class="fas fa-stars"></i>
-                  <span>{{ user.experience_points * 100 }} / {{ (user.experience_points + user.exp_needed) * 100 }} XP</span>
+                  <span>{{ user.experience_points * 100 }} / {{ (user.experience_points + user.exp_needed) * 100 }}
+                    XP</span>
```

Comment:
Seems like a weird line break to me

Reply:
You're right. Looks like autoformat didn't work too well on this one. I removed the linebreak